src/main/java/com/palantir/docker/compose/execution/DefaultDockerCompose.java

```diff
@@ -83,6 +83,36 @@ public void rm() throws IOException, InterruptedException {
     }
 
     @Override
+    public void exec(DockerComposeExecOption dockerComposeExecOption, String containerName,
+            DockerComposeExecArgument dockerComposeExecArgument) throws IOException, InterruptedException {
+        verifyDockerComposeVersion();
```

Comment:
This is a misleading method name. The better version will depend on what our implementation of Version looks like, but it will be something like:

verifyDockerComposeVersionAtLeast(Version.of(1.7.0))